cpp/src/slice2cs/Gen.cpp

```diff
@@ -1013,7 +1044,11 @@ Slice::Gen::TypesVisitor::visitClassDefEnd(const ClassDefPtr& p)
                 }
             }
         }
-        _out << parameters << epar;
+
+        // Primary constructor.
+        _out << sp;
+        writeDocLine(_out, "summary", "Initializes a new instance of the " + name + " class.");
```

Comment:
I didn't add doc-comment for the param constructors ... quite a bit of work, and not critical.